python/GafferUI/PathListingWidget.py

```diff
@@ -97,7 +98,7 @@ def __init__(
 		selectionMode = SelectionMode.Row,
 		displayMode = DisplayMode.List,
 		sortable = True,
-		horizontalScrollMode = GafferUI.ScrollMode.Never,
+		horizontalScrollMode = GafferUI.ScrollMode.Automatic,
```

Comment:
Technically this is an API breaking change that we should mention in Changes.md. Either that, or we change it back and then change the places where we're constructing a PathListingWidget. I think my general aversion to horizontal scrollbars clouds my judgement here, so I'll leave it to you...

Reply:
Makes sense. `Automatic` was useful for testing but I've reverted back to `Never` in 64482f7. There might be value in enabling `Automatic` on the HierarchyView once we start adding more columns, but I think we can leave this be for the moment...